docs/management/api/host-isolation-exceptions-api.asciidoc

```diff
@@ -0,0 +1,469 @@
+	"immutable": false,
+	"list_id": "{endpoint-artifact-list-id}",
+	"name": "Endpoint Security Host Isolation Exceptions List",
+	"namespace_type": "agnostic",
+	"os_types": [],
+	"tags": [],
+	"tie_breaker_id": "9894ae5c-2ce0-47dc-8c52-784ca2219fc3",
+	"type": "endpoint",
+	"updated_at": "2022-03-03T13:55:32.178Z",
+	"updated_by": "elastic",
+	"version": 1
+}
+--------------------------------------------------
+
+[#create-item-api]
+==== Create {endpoint-artifact-name-single}
+
+`POST <kibana host>:<port>/api/exception_lists/items`
+
+===== Request body
+
+A JSON object containing the fields listed below. The request must have the following:
+
+include::_exceptions-api-reusable-content.asciidoc[tags=endpoint-exceptions-request-payload-requirements]
+
+include::_exceptions-api-reusable-content.asciidoc[tag=create-exception-item-request-body]
+
+
+====== Example request
+
+
+[source,console,subs="attributes"]
+--------------------------------------------------
+POST api/exception_lists/items
+{
+  "description": "some description about this entry",
+  "entries": [
+      {
+          "field": "destination.ip",
+          "value": "10.10.0.1",
+          "type": "match",
+          "operator": "included"
+      }
+  ],
+  "list_id": "{endpoint-artifact-list-id}",
+  "name": "Some name for this item",
+  "namespace_type": "agnostic",
+  "os_types": [
+    "linux", 
+    "macos", 
+    "windows"
+  ],
+  "tags": [
+      "policy:all"
+  ],
+  "type": "simple"
+}
+--------------------------------------------------
+// KIBANA
+
+
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+===== Response payload
+
+[source,json,subs="attributes"]
+--------------------------------------------------
+{
+	"_version": "WzIxNDcsMV0=",
+	"created_at": "2022-03-03T14:04:00.496Z",
+	"created_by": "elastic",
+	"description": "some description about this entry",
+	"entries": [
+		{
+			"field": "destination.ip",
+			"value": "10.10.0.1",
+			"type": "match",
+			"operator": "included"
+		}
+	],
+	"id": "c6883700-9afa-11ec-94b1-fd7e90cc2a08",
+	"item_id": "1342081d-66ad-4b03-b554-131195ce404b",
+	"list_id": "endpoint_host_isolation_exceptions",
+	"name": "Host Isolation Exception - Created with API",
+	"namespace_type": "agnostic",
+	"os_types": [
+		"linux",
+		"macos",
+		"windows"
+	],
+	"tags": [
+		"policy:all"
+	],
+	"tie_breaker_id": "b65843e1-07b8-46d2-9e2b-5f0586dc364a",
+	"type": "simple",
+	"updated_at": "2022-03-03T14:04:00.498Z",
+	"updated_by": "elastic"
+}
+--------------------------------------------------
+
+
+
+[#update-item-api]
+==== Update {endpoint-artifact-name-single}
+
+`PUT <kibana host>:<port>/api/exception_lists/items`
+
+===== Request body
+
+A JSON object containing the fields listed below. The request must have the following:
+
+include::_exceptions-api-reusable-content.asciidoc[tags=endpoint-exceptions-request-payload-requirements]
+* `os_types` must be an array with all xref:exception-item-os-types[]
```

Comment:
Rephrase to match other items in list. Also, I think the `*` at the beginning of the line is causing it to render as a sub-bullet for some reason; changing to `-` might fix it.
```suggestion
- The `os_types` value must be an array with all xref:exception-item-os-types[].
```
<img width="500" alt="image" src="https://user-images.githubusercontent.com/87339667/156829399-9efc11ee-e0f7-4086-920f-a9e30cbd4bcd.png">